src/status_im/contexts/keycard/pin/enter/view.cljs

```diff
@@ -14,5 +14,4 @@
       {:icon-name :i/close
        :on-press  events-helper/navigate-back}]
      [quo/page-top {:title (i18n/label :t/enter-keycard-pin)}]
-     [rn/view {:style {:flex 1}}]
```

Comment:
so probably this was a reason somehow?

Reply:
and probably as i tested only only on one device, it might be a screen size was the reason

Reply:
I think the issue can be related to how the flex styling can squish other elements down. I've seen this happen before in the settings screens, and that was also caused by some weird flexbox issue.